Project.toml

```diff
@@ -18,30 +18,28 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [compat]
-ArgCheck = "1, 2.0"
-Combinatorics = "1.0"
-Compat = "3.0"
+Adapt = "1"
+ArgCheck = "1, 2"
+Compat = "3"
 Distributions = "0.23.3"
-DistributionsAD = "0.6"
-ForwardDiff = "0.10.3"
 MappedArrays = "0.2.2"
 NNlib = "0.6"
 Reexport = "0.2"
 Requires = "0.5, 1"
-Roots = "0.8.4, 1.0"
+Roots = "0.8.4, 1"
 StatsFuns = "0.8, 0.9.3"
-Tracker = "0.2.3"
-Zygote = "0.4.10"
 julia = "1"
 
 [extras]
+Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
```

Comment:
Where is `Adapt` used?

Reply:
It's a dependency of Tracker, it's never used explicitly. It's just a bugfix for the latest release (Tracker 0.2.7) which incorrectly specifies that it is compatible with Adapt 2 whereas actually it isn't (it was added but never tested since the checked-in Manifest.toml of Tracker was not updated...). There's some discussion in an open PR to the Tracker repo, I even suggested fixing the bounds in the registry (since it's just a patch release it's not possible to prevent the use of this latest release if the version is not specified explicitly, i.e., by `Tracker = "= 0.2.6"` etc.

The fix should be removed as soon as the problems in Tracker/Adapt are fixed.

Reply:
Ah, aight; sounds good! :+1: